src/ui/Toggle/__tests__/__snapshots__/Toggle.spec.js.snap

```diff
@@ -0,0 +1,26 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Toggle should do a snapshot test of the Toggle DOM 1`] = `
+<DocumentFragment>
+  <div
+    class="sendbird-ui-toggle undefined"
+  >
+    <label
+      class="sendbird-input-toggle-button  --unchecked"
```

Comment:
To better to follow BEM (atleast as we are trying to), for consistency, the classnames should be `sendbird-input-toggle-button  sendbird-input-toggle-button--unchecked `

Reply:
I was worried about the query selector became too longer in CSS